component/core/src/test/resources/conf/exo.social.component.core-configuration.xml

```diff
@@ -1687,6 +1722,707 @@
     </component-plugin>
   </external-component-plugins>
 
+  <external-component-plugins>
```

Comment:
Please move the definition of this component plugin in `exo.social.component.core-local-configuration.xml` to not having to affect all upper addons test configuration without needed it.